packages/core/src/app/order/OrderSummaryItem.tsx

```diff
@@ -46,7 +49,8 @@ const OrderSummaryItem: FunctionComponent<OrderSummaryItemProps> = ({
                 >
                     {productOptions.map((option, index) => (
                         <li className="product-option" data-test={option.testId} key={index}>
-                            {option.content}
+                            <span className="product-option-label">{option.optionLabel}</span>
```

Comment:
Maybe add a check here for content, just in case for customisations? Something like
```
{Boolean(content) ? 
    {option.content} : 
    <>
       <span className="product-option-label">{option.optionLabel}</span>
       <span className="product-option-value">{option.optionValue}</span>
    </>
 }
```